pkgs/top-level/python-packages.nix

```
@@ -5232,6 +5232,8 @@ in {
 
   twisted = callPackage ../development/python-modules/twisted { };
 
+  twitch-python = disabledIf (! pythonAtLeast "3.7") (callPackage ../development/python-modules/twitch-python { });
```

move into expression please
```suggestion
  twitch-python = disabledIf (! pythonAtLeast "3.7") (callPackage ../development/python-modules/twitch-python { });
```